src/main/java/com/amazon/ion/impl/bin/IonEncoder_1_1.java

```diff
@@ -509,34 +500,32 @@ public static int writeSymbolValue(WriteBuffer buffer, long value) {
      * Writes a Blob to the given WriteBuffer using the Ion 1.1 encoding for Ion Blobs.
      * @return the number of bytes written
      */
-    public static int writeBlobValue(WriteBuffer buffer, byte[] value) {
+    public static int writeBlobValue(WriteBuffer buffer, byte[] value, int start, int length) {
```

Comment:
🗺️ When I started implementing things, I realized that the lob methods needed `start` and `length`.